MANIFEST.in

```diff
@@ -1,3 +1 @@
 include README.md LICENSE.md CHANGELOG.md
-include src/sqlfluff/config.ini
-include src/sqlfluff/core/default_config.cfg
```

Comment:
These are handled in setup.cfg now via `[options.package_data]`. License is also added to `.whl` files via `license_files` metadata. `Manifest.in` is now only required for additional repo root level files we want included in the source dist.